### DISCUSSION TITLE
skip export apk is not valid

### DISCUSSION THREAD

**Discussion by utterances** [skiptools/skip#258] [Q&A] [answered]
maybe im missing something, I tried a successful skip export (1.1.8) produced apks but multiple android phones is complaining the apk is invalid... here's the apk, tested on a Samsung Galaxy A03s running Android 13.0
[SkipTestApp-release.apk.zip](https://github.com/user-attachments/files/17318690/SkipTestApp-release.apk.zip)

**Comment by marcprux** [comment]
Can you try the debug version? The release version will only be valid once it had been signed.

**Reply by utterances** [reply]
thanks, that worked, didn't know that about release APKs

**Reply by marcprux** [reply, marked as answer]
If you create `Android/app/keystore.jks` and `Android/app/keystore.properties` files, then the release artifact created from `skip export` will be signed with that key and should be installable on a device. See https://skip.tools/docs/deployment/#export-signing for more details.
